### PR TITLE
content: new pages modelli

### PR DESCRIPTION
**Pull request by Fupete**

### CODE REVIEW

File: src/data/content/modelli.yaml [hunk 1]
```diff
@@ -221,6 +224,23 @@ components:
...
+      # alt: Alt lorem ipsum #C
+      headingLevel: 3
+      background: medium # null | light | medium | dark
+      big: false
+      specular: true
+      subtitle: Il modello in via di definizione per il sito museale #C
+      buttons: #C #I
+        - label: Scopri il modello musei #C
+          btnStyle: primary
+          url: "musei/" #M
```
Comment (Fupete) [inline src/data/content/modelli.yaml:242]:
```suggestion
          url: "musei-civici/" #M
```

**Comment by Fupete** [comment]
@danielenole @danielaiozzo this is ready for me. Do your final review and we will merge it today. Thanks.

File: src/data/content/modelli/musei-civici.yaml [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+    - /siti-musei-civici
+    - /progetti/siti-web-musei
+    - /progetti/siti-web-musei-civici
+
+# ————————————————————————————————————————————
+# SEO, OPENGRAPH
+# ————————————————————————————————————————————
+seo:
+  name: Modello musei civici - Designers Italia #C REMIND "- Designers Italia" #AUTO
+  description: A breve tutte le risorse per realizzare con facilità esperienze digitali semplici e accessibili per i visitatori dei musei civici #C
```
Comment (danielaiozzo) [inline src/data/content/modelli/musei-civici.yaml:51]:
```suggestion
  description: Presto disponibili le risorse per realizzare con facilità esperienze digitali semplici e accessibili per i visitatori dei musei civici #C
```

File: src/data/content/modelli/musei-civici.yaml [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+# ————————————————————————————————————————————
+components:
+
+  # ///////////////////
+  # HERO
+  # ///////////////////
+  hero:
+    background: light
+    title: Musei civici #C
+    subtitle: A breve tutte le risorse per realizzare con facilità esperienze digitali semplici e accessibili per i visitatori dei musei civici #C
```
Comment (danielaiozzo) [inline src/data/content/modelli/musei-civici.yaml:100]:
```suggestion
    subtitle: Presto disponibili le risorse per realizzare con facilità esperienze digitali semplici e accessibili per i visitatori dei musei civici #C
```

File: src/data/content/modelli/musei-civici.yaml [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+    tag:
+        label: Modello #C
+        addonClasses: mt-3 text-uppercase
+    pretext:
+      icon:
+        icon: sprites.svg#it-info-circle
+        size: sm
+      text: In breve
+    text: | #C
+      Il modello, ora in lavorazione, ti permette di **realizzare siti dei musei civici** già validati dalla ricerca e progettati sui bisogni degli utenti, in modo rapido ed economico. Scopri tutti gli aggiornamenti sul modello, il processo di progettazione e le risorse presto disponibili.
```
Comment (danielaiozzo) [inline src/data/content/modelli/musei-civici.yaml:110]:
```suggestion
      Il modello, ora in lavorazione, ti permette di **realizzare siti dei musei civici** già validati dalla ricerca e progettati sui bisogni degli utenti, in modo rapido ed economico. Scopri tutti gli aggiornamenti sul modello, il processo di progettazione e le risorse in arrivo. 
```

**Review by danielaiozzo — CHANGES_REQUESTED** [review]
minor suggestions

**Review by danielaiozzo — APPROVED** [review]

**Review by danielenole — APPROVED** [review]

**Pull request merged**